src/mixins/transform-keys.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+import _ from 'lodash';
+
+function transformKeys(transformer) {
```

Comment:
@pandafox @jonelrempis @wildtron @wilbertverayin 
- I thought moving forward we will transition to snake case, so we should not add this in anymore?

Reply:
Depends on if we can consistently use snake casing throughout?

Reply:
I removed this for now